airflow/serialization/serialized_baseoperator.py

```diff
@@ -124,3 +142,50 @@ def _is_excluded(cls, var, attrname, op):
             # Don't store empty executor config or params dicts.
             return True
         return super()._is_excluded(var, attrname, op)
+
+    @classmethod
+    def _deserialize_operator_extra_links(
+        cls,
+        encoded_op_links: list
+    ) -> Dict[str, BaseOperatorLink]:
+        """
+        Deserialize Operator Links by importing all the modules containing operator links
+        defined in the serialized dict and initialize them by passing arguments using cattr.
+
+        :param encoded_op_links: Serialized Operator Link
+        :return: De-Serialized Operator Link
+        """
+
+        # Extra Operator Links
+        op_predefined_extra_links = {}
+
+        for _operator_links_source in encoded_op_links:
+            _operator_link_class, data = list(_operator_links_source.items())[0]
+            try:
+                single_link = import_string(_operator_link_class)
```

Comment:
I'm not sure if we should do this here (the import string) - my first thought was that we should look through the loaded plugins only.

Reply:
It is needed for the inbuilt OperatorLinks (BigQuery and Qubole) to work